internal/resources/cloudprovider/resource_aws_cloudwatch_scrape_job.go

```diff
@@ -96,7 +96,7 @@ func (r *resourceAWSCloudWatchScrapeJob) Schema(ctx context.Context, req resourc
 				Required:    true,
 			},
 			"regions": schema.SetAttribute{
-				Description: "A set of AWS region names that this CloudWatch Scrape Job applies to.",
+				Description: "A set of AWS region names that this CloudWatch Scrape Job applies to. This must be a subset of the regions that are configured in the associated AWS Account resource.",
```

Comment:
should we alter this to be called `regions_override`, and change the behaviour to allow null and empty values as we discused yesterday?